scripts/mbedtls_framework/c_parsing_helper.py

```diff
@@ -129,3 +153,17 @@ def read_function_declarations(functions: Dict[str, FunctionInfo],
                                        return_type,
                                        name,
                                        arguments)
+
+_C_TYPEDEF_DECLARATION_RE = r'typedef (?:struct ){0,1}(?P<type>\w+) (?P<name>\w+)'
```

Comment:
Why isn't the `struct` captured in `type`?

Reply:
Because we do not have to capture it to know it is a structure.